busylight/api/busylight_api.py

```diff
@@ -147,7 +190,7 @@ async def shutdown():
         logger.debug("problem during shutdown: {error}")
 
 
-## Exception Handlers
```

Comment:
Didn't like my markdown style comments? 😿

Reply:
Oh no! I do -- this must be another instance of my editor "auto-linting" things... sorry!